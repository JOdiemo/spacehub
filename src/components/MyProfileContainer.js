import RocketProfile from './RocketProfile';

const MyProfileContainer = () => (
  <div className="prof-container">
    <div className="rocketprofile">
      <h3 className="profile-title">My Rockets</h3>
      <RocketProfile />
    </div>
    <div className="missionprofile">
      <h3 className="profile-title">My Missions</h3>
      <RocketProfile />
    </div>
  </div>
);
export default MyProfileContainer;
