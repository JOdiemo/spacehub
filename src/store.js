import { configureStore } from '@reduxjs/toolkit';
import rocketReducer from './redux/Rockets/RocketsSlice';

const store = configureStore({
  reducer: {
    rocket: rocketReducer,
  },
});

export default store;
