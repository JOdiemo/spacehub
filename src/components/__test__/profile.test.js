import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';
import store from '../../store';
import MyProfileContainer from '../MyProfileContainer';

describe('Porfile component', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(
      <Provider store={store}>
        <Router>
          <MyProfileContainer />
        </Router>
      </Provider>,
    );
    expect(tree).toMatchSnapshot();
  });
});
