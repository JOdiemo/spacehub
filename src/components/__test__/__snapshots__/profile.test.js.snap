// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Porfile component matches the snapshot 1`] = `
<div
  className="prof-container"
>
  <div
    className="rocketprofile"
  >
    <h3
      className="profile-title"
    >
      My Rockets
    </h3>
    <div>
      <ul
        className="items"
      />
    </div>
  </div>
  <div
    className="missionprofile"
  >
    <h3
      className="profile-title"
    >
      My Missions
    </h3>
    <div>
      <ul
        className="items"
      />
    </div>
  </div>
</div>
`;
